project_event/i18n/fr.po

```diff
@@ -1256,7 +1257,7 @@ msgid "Tasks to Edit"
 msgstr "Tâches à modifier"
 
 #. module: project_event
-#: code:addons/project_event/models/project_task.py:883
+#: code:addons/project_event/models/project_task.py:892
 #, python-format
 msgid "Tasks: <br>"
 msgstr "Tâches: <br>"
```

Comment:
code:addons/project_event/models/project_task.py:892
#: code:addons/project_event/models/project_task.py:899
Refer to the same msgid "Tasks: <br>" . Is it possible to add single entry for both reference(both in same file)